lib/registries/addon/branded/route.ts

```diff
@@ -0,0 +1,20 @@
+import { action } from '@ember/object';
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import DS from 'ember-data';
+
+import Analytics from 'ember-osf-web/services/analytics';
+
+export default class BrandedRegistriesRoute extends Route {
+    @service analytics!: Analytics;
+    @service store!: DS.Store;
+
+    model(params: Record<string, string>) {
```

Comment:
```suggestion
    model(params: { provider_id: string }) {
```
`Record<string, string>` is a dangerous type because it implies that *any* property access will return a string, which can lead to errors that will only be caught at runtime
```ts
const foo: Record<string, string> = { blah: 'hello' };
foo.bleh.length; // type-checks fine, but throws TypeError at runtime
```

Reply:
fyi, if you're trying to write a type for an object that may have any set of string keys with string values, `Record<string, string | undefined>` might be a good fit, since it forces you to handle the case where a key doesn't exist.

but generally might be best to save `Record` for the cases when you have a constrained set of keys, to get the most of static typing:
```ts
type recordKeys = 'foo' | 'bar' | 'baz';
let myRecord: Record<recordKeys, string>;
myRecord = { foo: 'hi' }; // compile error: missing 'bar' and 'baz'
myRecord = { foo: 'a', bar: 'b', baz: 'c' }; // good
myRecord.blah; // compile error: 'blah' doesn't exist on myRecord
myRecord.baz.length; // good: myRecord.baz is reliably a string
```